src/scripts/main.js

```diff
@@ -1,3 +1,35 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+
+const listItems = [...tree.querySelectorAll('li')];
+
+listItems.forEach((elem) => {
+  const firstText = elem.firstChild;
+  const span = document.createElement('span');
+
+  span.textContent = `${firstText.textContent}`;
```

Comment:
The code assumes that `firstText` is a text node with a `textContent` property. If the first child is not a text node, this will result in an error. Consider checking if `firstText` is a text node before accessing `textContent`.